docs/README.md

```diff
@@ -0,0 +1,134 @@
+# Getting started with OpenTelemetry
```

Comment:
mm.. feels like "too much" information for "Getting started". The existing getting started (separate ones for console/host based) are truly getting-started style docs showing exactly what to do, instead of showing 2 possible ways and letting user pick one.

The content here is very good, but it should be placed some place other than getting started..(its awkward that this is all about initializing :D, but yet I ask to move it to someplace other than getting-started)

Reply:
If we speaking about moving to better place., maybe it is right time to move all docs to opentelemetry.io?

I think that it is the best place for all docs. The only downside is lack of CI for code posted there.

Reply:
Yes, but need to move all the docs, not just the content from *this* PR. There were previous attempts, but nothing got executed actually, so we are still with the undesired state of 2 places for the docs. (Pretty sure we have tracking issues for them)

Reply:
@cijothomas I went through and tried to clean things up. The new doc isn't part of getting started anymore or linked to from _anywhere_. I do plan to link to it, but I'll do as follow-ups.  I don't want to overdue it yet because the new API hasn't been released 🤣 Might be more of a post-1.10.0 effort, not sure yet. I moved the "getting started" content out of SDK README to the main one. Seemed to make way more sense there. Then just did misc cleanup for anything I saw giving it a read-through.

@Kielek 

> If we speaking about moving to better place., maybe it is right time to move all docs to opentelemetry.io?

I definitely don't have bandwidth to take this on! Feel free if you are passionate about it though. Personally I have mixed feelings about it. I think .NET devs expect this kind of documentation in the repo. What I would really love is a way to keep both places in sync automatically from a single source of truth (ideally the repo).